site/en/docs/extensions/mv3/match_patterns/index.md

```diff
@@ -38,8 +38,8 @@ Each part can use wildcards `*`. Below is a detailed description:
 `file:///`
 : Allows your extension to run on local files. It has no host and requires the user to manually [grant access][permissions-allow-access].
 
-Port URLs
-: Yuu can use `http://localhost/*` to match any localhost ports during development. For IP addresses, you can specify the address plus a wildcard in the path. For example: `http://127.0.0.1/*`
+Localhost URLs and IP addresses
+: To match any localhost port during development, use `http://localhost/*`. For example, it will match any paths under `http://localhost:8080` or `http://localhost:3000`. Another approach is to use `http://*:*/*` which will match `http://localhost:8080` and `https://example.com:my-path`. For IP addresses, you can use a broad host permissions `http://*/*"` or specify the address plus a wildcard in the path. For example: `http://127.0.0.1/*`. 
```

Comment:
I'm super confused by this (sorry):

* I find `http://*:*/*` makes perfect sense (I would've picked `http://localhost:*/*` as an example for how to match port - does that work?). However, I really don't understand why `http://localhost/*` matches the port. 
* I find `http://*/*"` a bad example as it basically matches everything
*  `https://example.com:my-path`` is not a valid URL

I'm sure I'm overthinking this way to much...

Reply:
(ㆆ_ㆆ)

- Both `http://localhost:*/*` and `http://localhost/*` work for localhosts... i picked the simple one.
- With the IPv4 addresses, the only pattern that worked was explicitly listing them or using `http://*/*`

I changed it again. If I still don't hit the target, I can remove this all together and go back to the drawing board later.